soul-common/src/test/java/org/dromara/soul/common/enums/DataSyncEnumTest.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.dromara.soul.common.enums;
+
+import org.junit.Test;
+
+import java.util.Arrays;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Test Cases for DataSyncEnumTest.
+ *
+ * @author patrickWuP
+ */
+public final class DataSyncEnumTest {
+    
+    @Test
+    public void testAcquireByName() {
+        Arrays.stream(DataSyncEnum.values()).forEach(e -> assertEquals(e, DataSyncEnum.acquireByName(e.getName())));
```

Comment:
remove this. 
```
Arrays.stream(DataSyncEnum.values()).forEach(e -> assertEquals(e, DataSyncEnum.acquireByName(e.getName())))
```

 can you add other DataSyncEnum test . for example :  ZOOKEEPER,WEBSOCKET

Reply:
get it